.github/workflows/static-analysis.yaml

```
@@ -0,0 +1,52 @@
+name: "Static Analysis"
+
+on:
+    pull_request:
+        branches:
+            - "[0-9]+.[0-9]+"
+            - "[0-9]+.x"
+    push:
+        branches:
+            - "[0-9]+.[0-9]+"
+            - "[0-9]+.x"
```

```suggestion
on:
    schedule:
        - cron: '0 3 * * 1,3,5'
    pull_request:
        branches:
            - "[0-9]+.[0-9]+"
            - "[0-9]+.x"
        paths-ignore:
            - 'doc/**'
            - 'src/Resources/public/**'
    push:
        branches:
            - "[0-9]+.[0-9]+"
            - "[0-9]+.x"
        paths-ignore:
            - 'doc/**'
            - 'src/Resources/public/**'
```

see also https://github.com/pimcore/ecommerce-framework-bundle/pull/39#discussion_r1172486327